src/backend/oracle-data-api/src/main/resources/data.sql

```diff
@@ -25,8 +25,15 @@ INSERT INTO JJDISPUTE (TICKET_NUMBER, CREATED_BY, CREATED_TS, MODIFIED_BY, MODIF
-  
+  ('EA00241528', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Francois Robert', 'Hodge', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', 'Bartwell Law & Associates', 'BARTWELL', 'BART', '', '', 'Polish', 4, 'IN_PERSON'),
+  ('EA07789632', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Chex', 'Mulligan', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', '', '', '', '', '', '', 0, 'IN_PERSON'),
+  ('EA01115462', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Hamza', 'Ben Adder', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', 'McGriff & Sons', 'McGriff', 'Dirk', '', '', 'Arabic', 4, 'IN_PERSON'),
+  ('EA04788953', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Yusuf', 'Mohammed', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', '', '', '', '', '', 'Arabic', 0, 'IN_PERSON'),
+  ('EA05888444', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Ho-Jong', 'Kim', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', '', '', '', '', '', 'Korean', 0, 'IN_PERSON'),
+  ('AO38375806', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Charles', 'LaRoque', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', '', '', '', '', '', 'French', 10, 'IN_PERSON'),
+  ('AU46117416', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Rebecca', 'Klein', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', '', '', '', '', '', '', 3, 'IN_PERSON'),
+  ('ST04605408', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Brian', 'McIntosh', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', 'Lional Hutz Storage Facility', 'Hutz', 'Lionel', '', '', '', 0, 'IN_PERSON');
+ 
```

Comment:
These ticket numbers don't match anything in our mock data (test-ticket-data.csv used by the ticket search).  This means that we have Disputes without matching JJDisputes and visa versa.

Reply:
These were the ones requested by business.